get-splice-junctions.wdl

```diff
@@ -2,13 +2,12 @@ version 1.0
 
 # ENCODE long read rna pipeline: get splice junctions
 
-#CAPER docker quay.io/encode-dcc/long-read-rna-pipeline:v1.3
-#CAPER singularity docker://quay.io/encode-dcc/long-read-rna-pipeline:v1.3
```

Comment:
You might want to add some usage documentation that `caper>=0.8.2.1` is required to autodetect docker image

Reply:
Good point, added to the installation instructions.